src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/introduction-opentelemetry-new-relic.mdx

```diff
@@ -159,46 +170,12 @@ APM agents offer detailed transaction trace visibility for individual services.
 </Callout>
 
-### Collector (recommended) [#collector]
-
-The OpenTelemetry project provides a tool called the [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector) that you can deploy and use as an intermediate data aggregator. In your service, you use the OpenTelemetry exporter to send telemetry data first to the OpenTelemetry collector. Then, in the OpenTelemetry collector, you enable the New Relic exporter to send data to New Relic.
-
-In this workflow, note the alternate path if you set up New Relic Infinite Tracing:
-
-![Diagram showing OpenTelemetry with collector](./images/collector_introduction_1.png "Diagram showing OpenTelemetry with collector")
-
-Here are some advantages of this approach:
-
-* The OpenTelemetry collector can perform processing operations on telemetry data before it is sent to the backend, including filtering attributes, excluding traces, and sampling.
-* The OpenTelemetry collector can receive telemetry data in many formats and forward it on to New Relic’s backend, including Prometheus, Zipkin, Jaeger, AWS X-Ray, OpenCensus, and much more.
-* The OpenTelemetry collector can collect system-level metrics from the host where it is running.
-
-### OTLP (pre-release) [#otlp]
-
-The example above uses a New Relic exporter, but we have a pre-release program if you want to try out the native OTLP endpoint for sending your data to New Relic. You can either use the OTLP exporter in the OpenTelemetry collector, or send us data directly from your service. If you are interested, let us know by completing this [form](https://docs.google.com/forms/d/e/1FAIpQLSdIJVEAYaP7TXe9LmQA64yIObGvt-nOiz5kXYsjxLBbvut_1A/viewform).
-
-Here are some advantages to operating without an OpenTelemetry collector:
-
-* There is no need to deploy a collector alongside your service.
-* Telemetry data is sent directly from the instrumented service to New Relic. If something goes wrong, and telemetry data does not arrive, there is one less component to debug.
-
-## What's next? [#what-is-next]
-
-If you want to set up OpenTelemetry with New Relic, the next step is to check out our [OpenTelemetry quick start](https://docs.newrelic.com/docs/integrations/open-source-telemetry-integrations/opentelemetry/set-opentelemetry-new-relic).
+Follow the [OpenTelemetry quick start](https://docs.newrelic.com/docs/integrations/open-source-telemetry-integrations/opentelemetry/set-opentelemetry-new-relic) to help you get started.
 
-If you prefer to use a New Relic APM agent, go to [Introduction to APM](https://docs.newrelic.com/docs/apm/new-relic-apm/getting-started/introduction-apm).
+You'll also want to review [tips](/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts) for getting the most out of the data you send to New Relic.
```

Comment:
this is a nitpick -> I would prefer saying "review our OpenTelemetry best practices" rather than "tips" since that's more aligned with what you're linking to

Reply:
@kaileyhaynes, good point! I just fixed this. Thanks!!